tests/bootstrap.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+define('APPLICATION_ROOT_DIR', __DIR__ . DIRECTORY_SEPARATOR . 'app' . DIRECTORY_SEPARATOR);
```

Comment:
What is this for exactly? :)

Reply:
https://spryker.atlassian.net/wiki/spaces/CORE/pages/2868248862/Standalone+module+testing